src/pages/DonateModal/DonateModal.tsx

```diff
@@ -27,36 +26,33 @@ export const DonateModal: React.FC<DonateModalProps> = ({ isVisible, onClose })
    * @description This hook is used to determine if the image should collapse or not based on if the user is a mobile, tablet user or not.
    */
   const shouldImgCollapse = useMediaQuery('(max-width: 950px)')
+  const { t } = useTranslation()
 
   return (
     <Modal
+      dir={i18n.dir()}
```

Comment:
is there a way to use global css rules to achive this behavior without inserting inline style to individual components?
If you don't know, I'd appreciate if you can open an issue to investigate it after we'll merge this one

Reply:
Not sure, Might be better to open an Issue about it but I can probably learn and do it alone

Reply:
I think it can be nice of you if you can do both (open an issue and assign yourself) - up to you :)